src/core/serializers/serializePlanInput.ts

```diff
@@ -144,6 +146,24 @@ export const serializePlanInput = (values: PlanFormInput) => {
             taxes: undefined,
           }
         : {},
+    usageThresholds: [
+      ...(nonRecurringUsageThresholds ?? []).map((threshold) => ({
+        ...threshold,
+        thresholdDisplayName: threshold.thresholdDisplayName ?? null,
```

Comment:
hum, If the user tries to edit the name, and then remove the value (at the same moment), the value will be an empty string if I'm correct
And there you will then send this empty string to the BE.
How would that be then on fetch? Will we receive `null` or `''`?
If it's null, maybe we should use `||` instead WDYT?